refinery/tool_manager/tests.py

```diff
@@ -3630,6 +3630,22 @@ def assertions(tool):
             assertions
         )
 
+    @override_settings(REFINERY_SOLR_DOC_LIMIT=10)
+    def test_input_node_limit(self):
+        tool = self.create_tool(ToolDefinition.VISUALIZATION)
+        tool_launch_config = self.tool.get_tool_launch_config()
+        tool_launch_config[Tool.FILE_RELATIONSHIPS] = str(
+                [uuid.uuid4()] * (10 + 1)
+            )
+        tool.set_tool_launch_config(tool_launch_config)
+        with self.assertRaises(VisualizationToolError) as context:
+            tool.launch()
+
+        self.assertIn(
+            "Input Node limit of: 10 reached",
+            context.exception.message
+        )
```

Comment:
This can fit on two lines instead of four

Reply:
Sounds like this issue again: https://github.com/refinery-platform/refinery-platform/issues/2675